refinery/data_set_manager/urls.py

```diff
@@ -45,7 +44,8 @@
     url(r'^import/take_ownership/$',
         login_required(TakeOwnershipOfPublicDatasetView.as_view()),
         name='take_ownership_of_public_dataset'),
-
+    url(r'^add-files/$', login_required(AddFilesToDataSetView.as_view()),
```

Comment:
add_files align with our other urls

Reply:
Align in what sense?

Reply:
our urls are snake_case

Reply:
OK. I didn't see anything about that mentioned in the coding style guide.
Also, Google says:
"We recommend that you use hyphens (-) instead of underscores (_) in your URLs."
https://support.google.com/webmasters/answer/76329

Reply:
I just double checked and we mostly use snake_case except for endpoints in data_set_manager related to imports. In two scoops, it recommends underscores to dashes. I believe we agree on using it as a guide for our code.

Reply:
Actually, Two scoops only talks about underscores in URL pattern names (section 1.6.2). A quick [Google search](https://www.google.com/search?q=url+hyphen+vs+underscore) also appears to suggest that hyphens are preferable to underscores in URLs. Perhaps it is time to set the standard?